source/elements/oneDAL/source/data_management/data_sources/index.rst

```diff
@@ -31,8 +31,11 @@ following:
 
 CSV data source
 -----------------
-Class ``csv::data_source`` is an implementation of a data source type
-for which the following is true:
+Class ``csv::data_source`` is an api for accessing the data source represented as csv file.
+A comma-separated values file (csv) is a text file. Each line of the csv file is a record, containing fields separated by the delimiter.
```

Comment:
does it make sense to add definition of csv into the glossary?

Reply:
Not sure that adding to glossary is necessary, since this will probably be the only place where we will use this term.